proto/storage/vulnerability.proto

```diff
@@ -56,6 +56,7 @@ message EmbeddedVulnerability {
   // cvss_metrics stores list of cvss scores from different sources like nvd, Redhat etc
   repeated CVSSScore cvss_metrics = 21;
   float nvd_cvss = 22; // @gotags: search:"NVD CVSS,store"
+  EPSS epss_score = 23;
```

Comment:
@charmik-redhat @ksurabhi91 What do y’all think if the property is `epss` for parallel form? That is,

```
EPSS epss = 23;
```